src/services/utilities.ts

```diff
@@ -1303,14 +1303,7 @@ namespace ts {
     export function getScriptKind(fileName: string, host?: LanguageServiceHost): ScriptKind {
         // First check to see if the script kind was specified by the host. Chances are the host
         // may override the default script kind for the file extension.
-        let scriptKind: ScriptKind;
-        if (host && host.getScriptKind) {
-            scriptKind = host.getScriptKind(fileName);
-        }
-        if (!scriptKind) {
-            scriptKind = getScriptKindFromFileName(fileName);

```

Comment:
is `host.getScriptKind` the same as `getScriptKindFromFileName` here? If not, where is `host.getScriptKind` defined? (I'm trying to figure out why you removed the fallback to `getScriptKindFromFileName`.)